rust-runtime/aws-smithy-http-server/src/protocols.rs

```diff
@@ -21,6 +22,7 @@ pub fn content_type_header_empty_body_no_modeled_input(headers: &HeaderMap) -> R
     }
 }
 
+#[allow(clippy::result_large_err)]
```

Comment:
since this is hand written, cc @awslabs/smithy-rs-server we might want to look into that

Reply:
Ticketed this up https://github.com/awslabs/smithy-rs/issues/2470